src/Dompdf.php

```diff
@@ -588,8 +588,10 @@ private function processHtml()
             switch (strtolower($tag->nodeName)) {
                 // load <link rel="STYLESHEET" ... /> tags
                 case "link":
-                    if (mb_strtolower(stripos($tag->getAttribute("rel"), "stylesheet") !== false) || // may be "appendix stylesheet"
-                        mb_strtolower($tag->getAttribute("type")) === "text/css"
+                    if (
+                        (stripos(mb_strtolower($tag->getAttribute("rel")), "stylesheet") !== false // may be "appendix stylesheet"
```

Comment:
Since the value is lower-cased, you can use `strpos` instead of `stripos`; or remove the call to `mb_strtolower`. But we can also change that later.

Reply:
Since the strings are ASCII-only, `mb_strtolower` could also be replaced by `strtolower`.

Reply:
You're right! Fixed